style.css

```diff
@@ -0,0 +1,32 @@
+// неправильный комментарий
+
+.hEad {
+
+}
+
+
+
+/*нет пробелов в комментарии, это ошибка*/
+
+.wrap {
+
+  position: ;
+
+  display: block;
+
+}
+
+.main {color: aliceblue}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/test_test/pullRequest?prid=5392485)